Tests/SonarScanner.MSBuild.PreProcessor.Tests/TargetsInstallerTests.cs

```diff
@@ -164,6 +164,43 @@ public void InstallLoaderTargets_Running_Under_LocalSystem_Account()
             act.Should().NotThrow();
         }
 
+        [TestMethod]
```

Comment:
The tests in this class need tidying up: there is quite a bit of duplication, and some of the old tests write directly to the file system.
I started refactoring them but it was it was starting to get complicated so I've just added this test for now to fix the code coverage issue.